js/models/wallet/Spend.js

```diff
@@ -118,7 +118,7 @@ export function spend(fields) {
   const attrs = {
     currency: app && app.settings && app.settings.get('localCurrency') || 'BTC',
     feeLevel: app &&
-    app.localSettings && app.localSettings.get('defaultTransactionFee') || 'PRIORITY',
+      app.localSettings && app.localSettings.get('defaultTransactionFee') || 'PRIORITY',
```

Comment:
I'm not sure why this updated. It was over the line character limit, but I don't think I have my IDE set to automatically fix that.